website/content/api-docs/secret/pki.mdx

```diff
@@ -1007,6 +1009,79 @@ $ curl \
 }
 ```
 
+### List Revocation Requests
+
+This endpoint returns a list of serial numbers that have been requested to
+be revoked on any cluster, along with information about the request's state
+and which cluster it originated on.
+
+| Method | Path                      |
+| :----- | :------------------------ |
+| `LIST` | `/certs/revocation-queue` |
+
+#### Sample Request
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request LIST \
+    http://127.0.0.1:8200/v1/pki/certs/revocation-queue
+```
+
+#### Sample Response
+
+```json
+{
+  "data": {
+    "key_info": {
+      "3d:80:91:c3:c2:34:3b:81:69:3d:92:a3:80:69:db:53:04:26:ab:b4": {
+        "requesting_cluster": "48327b28-8325-6d79-6a0b-4cbaa6f27b4a"
+      }
+    },
+    "keys": [
+      "3d:80:91:c3:c2:34:3b:81:69:3d:92:a3:80:69:db:53:04:26:ab:b4"
+    ]
+  }
+}
+```
+
+### List Cross-Cluster Revocations
+
+This endpoint returns a list of serial numbers that have been revoked on any
+cluster, along with a cluster that has a copy of that revoked certificate.
```

Comment:
nit:
```suggestion
cluster, along with the clusters that have a copy of that revoked certificate.
```

Reply:
I thought we decided that this API would only return a single cluster, not all clusters.

Reply:
That was the cross-cluster queue list, the revocation one returns [them all](https://github.com/hashicorp/vault/blob/e7636b47c6fd5261ab2f08d07971c23a0a6a937e/builtin/logical/pki/path_revoke.go#L622-L658).

Reply:
Ah, I see.